courses/management/commands/import_courserun.py

```diff
@@ -129,7 +129,7 @@ def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
                 )
                 return False
         elif kwargs["program"] is not None and kwargs["run_tag"] is not None:
-            for course in program.courses.all():
+            for course in program.courses:
```

Comment:
@jkachel have you tried running this?  I think this may need to handle the tuple returned from program.courses